plugins/UFM_NDT_Plugin/ufm_sim_web_service/topo_diff/ndt_infra.py

```diff
@@ -441,7 +525,12 @@ def create_topoconfig_file(links_info_dict, ndt_file_path, patterns,
                 topoconfig_file.write("%s,%s,%s,%s,%s,%s\n" % (port_guid, start_port,
                             peer_port_guid, peer_port,host_type,port_state))
     ndt_file.close()
-    return True
+#     ATB - no need to remove file
```

Comment:
why not to remove the file? 
if commented out lines are no longer relevant, please remove...

Reply:
no need to remove file. we still need it even was some issues ... may be we will deploy it as is